castai/resource_node_configuration.go

```diff
@@ -197,6 +197,11 @@ func resourceNodeConfiguration() *schema.Resource {
 							Optional:    true,
 							Description: "Allow IMDSv1, the default is true",
 						},
+						"imds_hop_limit": {
+							Type:        schema.TypeInt,
+							Optional:    true,
+							Description: "Allow configure the IMDSv2 hop limit, the default is 2",
```

Comment:
Maybe let's set default also in terraform? Like for example cpuDiskratio?

Reply:
done